multisig/proposals/opbnbtestnet/vip-018/index.ts

```diff
@@ -0,0 +1,50 @@
+import { parseUnits } from "ethers/lib/utils";
+
+import { makeProposal } from "../../../../src/utils";
+
+const XVS_BRIDGE_ADMIN_PROXY = "0x19252AFD0B2F539C400aEab7d460CBFbf74c17ff";
+export const SINGLE_SEND_LIMIT = parseUnits("10000", 18);
+export const MAX_DAILY_SEND_LIMIT = parseUnits("50000", 18);
+export const SINGLE_RECEIVE_LIMIT = parseUnits("10000", 18);
+export const MAX_DAILY_RECEIVE_LIMIT = parseUnits("50000", 18);
```

Comment:
Receive limits should be 2% greater than "sending" limits, to avoid blockers in the bridge process if the XVS price changes significantly during the offchain process of a transfer. So, the receive limits would be 10,200 and 51,000

Reply:
Done https://github.com/VenusProtocol/vips/pull/249/commits/e88d273c9f35b498b210078d704a5be113fb0daa